features/parse-html-unsafe.yml.dist

```diff
@@ -2,14 +2,11 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
-  baseline_low_date: 2024-04-18
+  baseline: false
   support:
     chrome: "124"
     chrome_android: "124"
     edge: "124"
-    firefox: "123"
-    firefox_android: "123"
```

Comment:
This due to https://github.com/mdn/browser-compat-data/pull/23401, where `api.Document.parseHTMLUnsafe_static` is broken before Firefox 128. When 128 actually ships, this will get updated again (about July 9).